app/views/spree/admin/overview/single_enterprise_dashboard.html.haml

```diff
@@ -47,14 +47,15 @@
     .header
       %h3
         %span.icon-map-marker
-        Your profile live
-      %p on the Open Food Network map
+          = t "your_profil_live"
+      %p
+        = t "on_ofn_map"
     .list
       /-# Can we pass an anchor here to zoom to our enterprise?
       %a.button.bottom{href: main_app.map_path, target: '_blank'}
-        See
+        = t "see"
```

Comment:
I read haml reference and the good format for internationalisation is : 
t('#name'')
Sometimes we find t('#name'') or t '#name'' or t :name(are u sure it works well?), we need to define a template...